src/mod/filesys.mod/filesys.c

```diff
@@ -805,8 +805,8 @@ static void filesys_dcc_send_hostresolved(int i)
       open_telnet_raw(dcc[i].sock, &dcc[i].sockname) < 0)
         dcc[i].type->eof(i);
 #ifdef TLS
-      else if (dcc[i].ssl && ssl_handshake(dcc[i].sock, TLS_CONNECT, tls_vfydcc,
-                                           LOG_FILES, dcc[i].host, NULL))
+      else if (dcc[i].ssl & DCC_TLS_USE && ssl_handshake(dcc[i].sock,
```

Comment:
paranthesis around (x & y) && z are probably recommended by the compiler as a warning :)